docs/source/references/config.md

```diff
@@ -124,40 +124,40 @@ Has type `BlockSyncConfiguration`. Can be configured via environment variable `I
 
 ```json
 {
-  "BATCH_SIZE": 4,
-  "GOSSIP_PERIOD_MS": 10000,
-  "MAILBOX": 100
+  "ACTOR_CHANNEL_CAPACITY": 100,
+  "BLOCK_BATCH_SIZE": 4,
+  "GOSSIP_PERIOD_MS": 10000
 }
 ```
 
-### `block_sync.batch_size`
+### `block_sync.actor_channel_capacity`
 
-The number of blocks, which can be sent in one message.
+Actor channel capacity
 
-Has type `u32`. Can be configured via environment variable `BLOCK_SYNC_BATCH_SIZE`
+Has type `u32`. Can be configured via environment variable `BLOCK_SYNC_ACTOR_CHANNEL_CAPACITY`
 
 ```json
-4
+100
 ```
 
-### `block_sync.gossip_period_ms`
+### `block_sync.block_batch_size`
 
-The time between sending request for latest block.
+The number of blocks, which can be sent in one message.
 
-Has type `u64`. Can be configured via environment variable `BLOCK_SYNC_GOSSIP_PERIOD_MS`
+Has type `u32`. Can be configured via environment variable `BLOCK_SYNC_BLOCK_BATCH_SIZE`
 
 ```json
-10000
+4
 ```
 
-### `block_sync.mailbox`
+### `block_sync.gossip_period_ms`
 
-Mailbox size
+The time between sending request for latest block.
```

Comment:
`between` implies there should be two events. do you mean the time between requests?